airqo-platform/AirQo-frontend
# [Web] hardcode Engagement Graduate Trainee posting

#### Summary of Changes (What does this PR do?)
- Add Engagement Graduate Trainee posting

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch and checkout to this branch locally
* Follow instructions in the README.md file to setup and run the app.
* Confirm the summary of changes listed above

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)
